xml_inscriptions/transcribed/MI.AA.UM.KM.L.1000.xml

```diff
@@ -63,6 +63,9 @@
                         CE</date>
                      <placeName ref="europe.italy.puteoli">Italy, Pozzuoli</placeName>
                   </origin>
+                  <provenance type="transferred" subtype="purchased">
+                     <p>Purchased from the collection of <persName>Guiseppe De Criscio</persName> for the <orgName role="recipient">University of Michigan</orgName> (Tuck).</p>
```

Comment:
Might want to add `@role` for De Criscio? Can probably be done as a bulk change after the fact, but just something to keep in mind.